molecule/archive-community-plaintext-rhel/molecule.yml

```diff
@@ -77,11 +77,13 @@ platforms:
   # Control center not included in community archive, must not have c3 in inventory
 provisioner:
   name: ansible
+  env:
```

Comment:
this env var might not be necessary with latest molecule

Reply:
I just confirmed this env var is not needed with my molecule version 
```
% molecule --version
molecule 3.3.0 using python 3.8
    ansible:2.11.1
    delegated:3.3.0 from molecule
    docker:0.2.4 from molecule_docker
```

I vaguely remember needing it in previous molecule version. Do you think we should have it/ be redundant or remove it and potentially force ppl to upgrade molecule?

Reply:
If it is required in older versions of molecule, I think we can/should keep it. As long as it still works in newer versions.

Reply:
Looks like jenkins is using molecule 3.0 and the env var is not required
```
12:36:31  + molecule --version
12:36:31  molecule 3.0.2.1
12:36:31     ansible==2.9.21 python==2.7
```
should remove that env var

Reply:
Correction, you need that env var for molecule 3.0, probably safest to keep it then:
```
[31mERROR! the role 'confluent.test.ldap' was not found in /home/jenkins/workspace/c_Contributors_cp-ansible_PR-701/molecule/rbac-scram-custom-rhel/../roles:/home/jenkins/.cache/molecule/c_Contributors_cp-ansible_PR-701/rbac-scram-custom-rhel/roles:/home/jenkins/workspace:/home/jenkins/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/jenkins/workspace/c_Contributors_cp-ansible_PR-701/molecule/rbac-scram-custom-rhel/..[0m
```

Reply:
Correction, you need that env var for molecule 3.0, probably safest to keep it then:
```
[31mERROR! the role 'confluent.test.ldap' was not found in /home/jenkins/workspace/c_Contributors_cp-ansible_PR-701/molecule/rbac-scram-custom-rhel/../roles:/home/jenkins/.cache/molecule/c_Contributors_cp-ansible_PR-701/rbac-scram-custom-rhel/roles:/home/jenkins/workspace:/home/jenkins/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/jenkins/workspace/c_Contributors_cp-ansible_PR-701/molecule/rbac-scram-custom-rhel/..[0m
```

Reply:
can consider for follow up work